base/src/environment/environment.ts

```diff
@@ -102,7 +102,7 @@ export class Environment implements ZLUX.Environment {
           else { //TODO what happens when there are multiple values
             const host = cache.userEnvironment.ZWED_node_mediationLayer_server_hostname
               ? cache.userEnvironment.ZWED_node_mediationLayer_server_hostname
-              : cache.userEnvironment.ZOWE_EXPLORER_HOST;
+              : cache.userEnvironment.ZWE_zowe_externalDomains_0;
```

Comment:
Would it be useful here to add a check for internal host incase external host isn't available i.e.

`cache.userEnvironment.ZWE_zowe_externalDomains_0 || cache.userEnvironment.ZWE_haInstance_hostname`